test/api/layout/execute_layout_test.cpp

```diff
@@ -321,6 +321,104 @@ TEST(execute_test, many_ubs)
                                       "#LOWER_LEVEL_IBF_26 fullest_technical_bin_idx:0\n"
                                       "#LOWER_LEVEL_IBF_27 fullest_technical_bin_idx:0\n"
                                       "#USER_BIN_IDX\tTECHNICAL_BIN_INDICES\tNUMBER_OF_TECHNICAL_BINS\n"
+#ifdef _LIBCPP_VERSION // seqan::hibf::sketch::toolbox::sort_by_cardinalities is not stable
```

Comment:
but it is deterministic within the same compiler?

what a code bloat :sweat:

Reply:
It's deterministic/constitent within each stl (libc++/libstdc++).

The alternative would be to use:
* `std::ranges::stable_sort` in the HIBF method, but this has a potentially higher complexity, and also needs additional memory
* make each size distinct. E. G. 200,201,202,..

Reply:
nah. it's fine.